Felienne/hedy
# [BUG FIX] Single achievement verification is working again

**Description**
Due to the re-structuring of the achievements YAML the single achievement pop-up was broken. We fix this in this PR with a very small addition to an if-statement. We also fix an issue where teacher achievements where not hidden from non-teacher accounts as well as hidden achievements being shown when not achieved. Lastly we fix the dutch achievements being incorrectly translated.

**Fixes**
This PR fixes #2747.

**How to test**
Make sure you are a teacher, create a class and verify that you receive the correct achievement. Make sure you don't have this achievement already.
